library/src/main/java/com/mapzen/android/DataLayerType.java

```diff
@@ -0,0 +1,17 @@
+package com.mapzen.android;
+
+/**
+ * Created by sarahlensing on 7/27/16.
+ */
+public enum DataLayerType {
```

Comment:
I was on the verge of doing this during the first pass but was slightly swayed not to because some `PersistableMapData` objects represent two different layer types (START_END_PIN has "mz_route_start" and "mz_route_destination". TRANSIT_ROUTE_LINE_STATION_ICON has "mz_route_line_transit" and"mz_route_transit_stop" data layers). I also wanted the switch statement where we restore data to be exhaustive but I can definitely see the convenience of maintaining only one list. I'll update.